src/python/install.sh

```diff
@@ -449,12 +453,17 @@ fi
 
 # Install JupyterLab if needed
 if [ "${INSTALL_JUPYTERLAB}" = "true" ]; then
-    install_user_package jupyterlab
+    if [[ $(python --version) != "" ]]; then
```

Comment:
Can you use the binary at `$PYTHON_SRC` for the check here instead of `which python`? `install_user_package` will use the binary at `$PYTHON_SRC`

Reply:
Good point.